apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -68,12 +74,24 @@ const Input = <T extends FieldValues>({
 						))}
 					</div>
 				) : (
-					<input
-						className={getValidClassNames(styles["input-field"])}
-						{...field}
-						placeholder={placeholder}
-						type={type}
-					/>
+					<>
+						<input
+							className={getValidClassNames(styles["input-field"])}
+							{...field}
+							placeholder={placeholder}
+							type={type}
+						/>
+						{iconName && (
+							<Button
+								hasVisuallyHiddenLabel
+								iconName={iconName}
+								label=""
```

Comment:
Add a label here. It should be added for accessibility purposes

Reply:
Can i add label like label="Icon" ?